cypress/integration/email-notification-form-spec.js

```diff
@@ -0,0 +1,215 @@
+        .get('.submit-button').click()
+        .wait('@successfulPost')
+        .get('input[id=first_name]').should('be.empty')
+        .get('input[id=last_name]').should('be.empty')
+        .get('select[id=state_name]').should('have.value', null)
+        .get('input[id=email]').should('be.empty')
+        .get('input[id=english]').should('not.be.checked')
+        .get('input[id=spanish]').should('not.be.checked')
+        .get('input[id=agree_to_emails]').should('have.value', 'false')
+    })
+  });
+
+  it('Should display a loading image while the submitting the form', () => {
+    cy.intercept('POST', 'http://localhost:3001/api/v1/users', (req) => {
+      req.reply({
+        delay: 1000,
+        fixture: 'success.json'
+      })
+    })
+
+    cy.fixture('user3.json').as('user3').then((user3) => {
+      cy.get('input[id=first_name]').type(user3.first_name)
+        .get('input[id=last_name]').type(user3.last_name)
+        .get('select[id=state_name]').select(user3.state_name)
+        .get('input[id=email]').type(user3.email)
+        .get('input[id=english]').click()
+        .get('input[id=agree_to_emails]').click()
+        .get('.submit-button').click()
+        .get('.loading-icon').should('be.visible')
+    })
+  });
+
+  it('Should display an error message if the email is already subscribed', () => {
+    cy.intercept('POST', 'http://localhost:3001/api/v1/users', (req) => {
+      req.reply({
+        fixture: 'errorDuplicate.json'
+      })
+    }).as('duplicatePost')
+
+    cy.fixture('user1.json').as('user1').then((user1) => {
+      cy.get('input[id=first_name]').type(user1.first_name)
+        .get('input[id=last_name]').type(user1.last_name)
+        .get('select[id=state_name]').select(user1.state_name)
+        .get('input[id=email]').type(user1.email)
+        .get('input[id=english]').click()
+        .get('input[id=agree_to_emails]').click()
+        .get('.submit-button').click()
+        .wait('@duplicatePost')
+        .get('.server-message').should('contain', `${user1.email} is already subscribed to receive election notifications.`)
+    })
+  });
+
+  it('Should display an error image & error message if the server can\'t complete the request', () => {
+    cy.intercept('POST', 'http://localhost:3001/api/v1/users', {statusCode: 500}).as('getServerFailure')
+
+    cy.fixture('user1.json').as('user1').then((user1) => {
+      cy.get('input[id=first_name]').type(user1.first_name)
+        .get('input[id=last_name]').type(user1.last_name)
+        .get('select[id=state_name]').select(user1.state_name)
+        .get('input[id=email]').type(user1.email)
+        .get('input[id=english]').click()
+        .get('input[id=agree_to_emails]').click()
+        .get('.submit-button').click()
+        .wait('@getServerFailure')
+    })
+    cy.get('.error-text').should('contain', 'We\'re sorry, please try again.')
+  });
+
+  xit('Should display a button to send the user back to the form', () => {
+
+  });
```

Comment:
I am going to leave it for now, as it will be used once router is implemented and a button can be added.  I'm sure my brain won't let me forget about it, but it's one more line of defense against forgetting to test that functionality. 😅

Reply:
We can always come back and add more tests like this one. For now, I think it would be good to remove it since we don't want it to interfere with our ability to implement Continuous Integration and Deployment.